client/app/pages/dashboards/dashboard.js

```diff
@@ -178,6 +179,7 @@ function DashboardCtrl(
       { slug: $routeParams.dashboardSlug },
       (dashboard) => {
         this.dashboard = dashboard;
+        $scope.dashboard = dashboard;
```

Comment:
This seems kind of ugly, `$scope` wasn't used in this file up until now. I haven't found any other way to be able to use the dashboard id in the angular template for the dashboard though.

Reply:
`$ctrl.dashboard.id`, no?

Reply:
Does not seem to work when I use it here: https://github.com/getredash/redash/pull/2890/files#diff-374cafdf3778c7e0169295ee5878667bR66

Reply:
This does in fact work - it was a problem with my IDE, which incorrectly highlighted {{ $ctrl.dashboard.id }} as a syntax error. Thank you 👍